aten/src/ATen/Declarations.cwrap

```diff
@@ -812,34 +714,6 @@
         - THTensor* other
 ]]
 [[
-  name: THPTensor_(apply)
-  python_name: apply_
-  defined_if: "!IS_DISTRIBUTED"
-  backends:
-    - CPU
-  cpu_half: True
-  only_register: True
-  override_method_flags: METH_O
-]]
-[[
-  name: THPTensor_(map)
-  python_name: map_
-  defined_if: "!IS_DISTRIBUTED"
-  backends:
-    - CPU
-  cpu_half: True
-  only_register: True
-]]
-[[
-  name: THPTensor_(map2)
-  python_name: map2_
-  defined_if: "!IS_DISTRIBUTED"
-  backends:
-    - CPU
-  cpu_half: True
-  only_register: True
-]]
```

Comment:
What happened with all those functions? I doubt `map` and `map2` is implemented in ATen

Reply:
They still exist in `torch/csrc/generic/methods/TensorApply.cwrap` and co. AFAICT, there is some duplication between these, and ATen doesn't make use of these at all.